<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="cacheBox">
       <property name="title">
        <string>Base</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="defaultWorkspaceAsLabel">
              <property name="text">
               <string>Default Workspace As:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="filePanelRadioButton">
              <property name="text">
               <string>File Panel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="textEditorRadioButton">
              <property name="text">
               <string>Text Editor</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="homePathAsDefaultPathcheckBox">
            <property name="text">
             <string>Use home path as the default path for FilePanel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="clearGpgPasswordCacheCheckBox">
            <property name="text">
             <string>Clear gpg password cache when closing GpgFrontend.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="restoreTextEditorPageCheckBox">
            <property name="text">
             <string>Cache text editor contents.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="importConfirmationBox">
       <property name="title">
        <string>Operation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="importConfirmationCheckBox">
            <property name="text">
             <string>Import key files dropped on the Key List without confirmation.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="disableLoadingModulesCheckBox">
            <property name="text">
             <string>Disable loading of all modules (including integrated modules)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="langBox">
       <property name="title">
        <string>Language</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="langSelectBox">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="langNoteLabel">
            <property name="text">
             <string>NOTE: GpgFrontend will restart automatically if you change the language!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="dataBox">
       <property name="title">
        <string>Data</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="revealInFileExplorerButton">
            <property name="text">
             <string>Reveal in File Explorer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearAllLogFilesButton">
            <property name="text">
             <string>Clear All Log Files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearAllDataObjectsButton">
            <property name="text">
             <string>Clear All Data Objects Files</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
